addons/purchase/controllers/portal.py

```diff
@@ -129,11 +141,31 @@ def portal_my_requests_for_quotation(self, page=1, date_begin=None, date_end=Non
             "/my/rfq",
             'my_rfqs_history',
             'rfq',
-            'rfqs'
+            'rfqs',
+            self._rfq_get_searchbar_inputs(),
+            search_domain,
+            search,
+            search_in,
         )
 
+    def _rfq_get_searchbar_inputs(self):
+        values = {
+            'name': {'input': 'name', 'label': _('Search in Name'), 'order': 1},
+        }
+        return dict(sorted(values.items(), key=lambda item: item[1]["order"]))
```

Comment:
Why so much complexity ?

Reply:
cuz there were more search filters before, will simplify this